crates/astria-composer/src/collectors/geth.rs

```diff
@@ -82,8 +83,14 @@ impl Geth {
     /// Initializes a new collector instance
     pub(crate) fn new(chain_name: String, url: String, executor_handle: executor::Handle) -> Self {
         let (status, _) = watch::channel(Status::new());
+        let rollup_id = RollupId::from_unhashed_bytes(&chain_name);
+        info!(
```

Comment:
nit: I prefer after-the-fact event messages like "I have done the thing" over "I am going to do the thing" especially for short, non-async constructors.